### PR TITLE
Nataliia Vyrsta

### PR DESCRIPTION
**Pull request by NVyrsta**
- [DEMO LINK](https://NVyrsta.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_reset.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```
Comment (Malva37) [inline src/styles/_reset.scss:11]: try to avoid added this property to "*"

**Comment by Malva37** [comment]
good job))
you can create separate folder "utils" for reset & variables files https://prnt.sc/qwlGSVvPbo03

**Review by Malva37 — CHANGES_REQUESTED** [review]
good job, you have to fix a little)

File: src/styles/_reset.scss (hunk 1 above)
Comment (NVyrsta) [reply src/styles/_reset.scss:11]: thanks)

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**